src/transformers/models/idefics/vision.py

```diff
@@ -380,6 +425,7 @@ class IdeficsVisionTransformer(nn.Module):
     def __init__(self, config: IdeficsVisionConfig):
         super().__init__()
         self.config = config
+        self.interpolate_pos_encoding = config.interpolate_pos_encoding
```

Comment:
To align with other models, this should be a kwarg in the forward method, rather than set by the config

Reply:
Indeed that makes more sense